rails-homework/app/views/physicians/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with model: @physician do |form| %>
```

Comment:
As I know you can use `form_for` here and pass only `@physician` variable. Also please read about the difference between.